share/spice/quandl/fundamentals/quandl_fundamentals.css

```diff
@@ -0,0 +1,74 @@
+/*
+    Document   : 
+    Created on : 2015-02-12 08:21 PM
+    Author     : brianrisk
+    Description: styles based on the 'Stocks' spice for visual consistency
+*/
+
+#quandl h1.quandl-header {
```

Comment:
@brianrisk understood, thanks for explaining.

@sdougbrown any thoughts on this? I don't believe the ID selector will cause problems because the div is created by the template

One alternative is to use a CSS selector like:

```csss
div[class^='zci--quandle'] {
    ...
}
```